docs/modules/wan/pages/wan.adoc

```diff
@@ -38,9 +38,9 @@ events and replicates them actively to other clusters.
 * Passive cluster: The user updates performed on this cluster are not replicated to other clusters. In other words,
 this cluster can be seen as the "target" cluster which is capable of receiving, applying and possibly forwarding
 WAN events from other ("active") clusters. It does not generate any WAN update events because of user interaction.
-* WAN publisher: A publisher is a sink for WAN events and an implementation of `WanReplicationPublisher`.
-Most often, this is a single, entire target Hazelcast cluster but you can also define custom publishers which
-may transmit WAN events to other systems such as messaging queues, Kafka or even persist events on disk.
+* WAN publisher: A publisher acts as a sink for local WAN events and pushes them to a target. The target is usually
+another Hazelcast cluster, but can be customized to be another system, such as a messaging queue, Kafka or even persisted
```

Comment:
The target is usually another Hazelcast cluster; however, you could also use a different target such as a message queue, Kafka, or saving data to persistent storage.

Reply:
Questions not related to the update, but as I looked and saw it....

1. Can we link to any docs on WanReplicationPublisher? If not, what does it add to tell me that this is an implementation of it?
2. Can we link to any information on using different targets? Possibly the [Configuring WAN Replication ](https://docs.hazelcast.com/hazelcast/5.3/wan/defining-wan-replication)page, and/or [Advanced Features](https://docs.hazelcast.com/hazelcast/5.3/wan/advanced-features)?

Reply:
Thanks @rebekah-lawrence, your tweaks read better. 

@JamesHazelcast or @vbekiaris may have some suggestions about informative links.

Reply:
"different targets" like JMS etc are examples of custom `WanPublisher` implementations a user can implement, it's not something we support out of the box. To be honest, I have never seen a user/customer implement their own `WanPublisher`.
The `WanReplicationPublisher` name is probably outdated - the interface is called `WanPublisher` and it's what users should implement to provide their custom WAN publisher implementation. Probably the audience here would be a java developer, so I think it would be most appropriate to link to `WanPublisher` [javadoc](https://docs.hazelcast.org/docs/latest-dev/javadoc/com/hazelcast/wan/WanPublisher.html)